packages/scaffolding-cli/templates/src/ssr/environment-configuration/environment-variables.js

```diff
@@ -7,4 +7,10 @@ module.exports = {
     LOG_LEVEL: process.env.LOG_LEVEL,
     POD_NAME: process.env.POD_NAME,
     APPINSIGHTS_KEY: process.env.APPINSIGHTS_INSTRUMENTATIONKEY,
+
+    AUTH0_BASE_URL: `${process.env.APP_BASE_URL}:${process.env.PORT}`,
+    AUTH0_CLIENT_ID: process.env.AUTH0_CLIENT_ID,
+    AUTH0_DOMAIN: process.env.AUTH0_DOMAIN,
+    AUTH0_CLIENT_SECRET: process.env.AUTH0_CLIENT_SECRET,
```

Comment:
Should the client secret be public? Or should it be part of the server runtime?

Reply:
Yes. Good shout. I have seperated them out now.